### DISCUSSION TITLE
Make Stats more Persistent in 0.14

### DISCUSSION THREAD

**Discussion by accessiblepixel** [blakeblackshear/frigate#14159] [Ask A Question] [answered]
Hi! I had a look in the discussions and issues to see if I could see any references to it, and some internet searches.

Is it possible to keep the stats page statistics persistent (the system#general page) across docker restarts/rebuilds?

I looked under the docker docs to see if there was a path I could map to a volume but couldn't find one.

Is this possible? :) Thanks

**Comment by NickM-27** [comment, marked as answer]
No,and it's not designed for this. It's designed to give an overall picture of current system health

**Reply by accessiblepixel** [reply]
Hmm. Okay. It just never seems to fill the graph, figured it was a persistence thing I needed to set, if I leave it open it fills half way and then scrolls onward... like I've had this open for half hour (without a restart or rebuild) and it's constantly only filled the graphs less than half way, does it not go fully across at any point?

![halff hourr](https://github.com/user-attachments/assets/2f17ccdc-eda2-45cf-a760-d75701243ac2)

^ looks like this regardless of when I open it, or how long I open it.

Thanks

**Reply by NickM-27** [reply]
That might be a bug or other issue, it fills all the way for me

**Reply by NickM-27** [reply]
Have you tried fully refreshing the page?

**Reply by accessiblepixel** [reply]
> Have you tried fully refreshing the page?

I hadn't tried a forced refresh since it looked like it was polling automatically/correctly. Now it shows all the graph and I feel silly for not trying that. Thank you. <3
